arc/scenarios/quality_scorer.py

```
@@ -54,14 +68,21 @@ def __init__(self, min_threshold: float = 3.0):
             "raises", "encounters", "experiences", "suffers from"
         ]
     
-    def score_scenario(self, scenario: Dict[str, Any]) -> QualityMetrics:
+    def score_scenario(self, scenario: Union[Scenario, Dict[str, Any]]) -> QualityMetrics:
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)